src/components/map/Map.tsx

```diff
@@ -74,40 +76,40 @@ const MapIcon = ({
   return <MarkerIcon color={iconColor} size={iconSize} />;
 };
 
-const renderCluster = (cluster: TCluster) => {
-  const { clusterColor: backgroundColor, geometry, id, onPress, properties = {} } = cluster;
-  const { point_count } = properties;
+const renderCluster = ({ markers, onPress }: { markers: any[]; onPress: () => void }) => {
+  let clustersData = {
+    features: [],
+    type: 'FeatureCollection'
+  };
+
+  markers.map((item) => {
+    if (item.properties.point_count > 0) {
+      clustersData.features.push(item);
+    }
+  });
 
   return (
-    <PointAnnotation
-      coordinate={geometry.coordinates}
-      id={`cluster-${id}`}
-      key={`cluster-${id}`}
-      onSelected={onPress}
-      style={styles.hitBox}
+    <ShapeSource
+      hitbox={{ width: HIT_BOX_SIZE, height: HIT_BOX_SIZE }}
```

Comment:
why do we need to override the default of 44x44?

Reply:
Because our hitbox size is not 44x44 but instead `normalize(50)`